examples/c/template_example.c

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
I guess you just added this to compare results with the templated code.
Maybe unnecessary to put in the repo.
I think we should rather reduce the number of examples and maintain the remaining.

Reply:
Indeed. We can take it out ;)